scripts/zones/Norg/Zone.lua

```diff
@@ -7,11 +7,13 @@ local ID = require("scripts/zones/Norg/IDs")
 require("scripts/globals/conquest")
 require("scripts/globals/keyitems")
 require("scripts/globals/missions")
+require("scripts/globals/quests")
 require("scripts/globals/chocobo")
 -----------------------------------
 
 function onInitialize(zone)
     tpz.chocobo.initZone(zone)
+	zone:registerRegion(1, -24, 0, -59, -15, 1, -50)  -- Near the SSG exit
```

Comment:
Here, and in some of the other files, you've got tabs.  Project standard is 4 spaces.  There's probably a setting in your text editor to automatically replace tabs with 4 spaces.